Added stuff

josheewa opened the pull request.

josheewa committed:
Merge pull request #4 from Legend-of-iPhoenix/master

update

josheewa committed:
Merge pull request #5 from Legend-of-iPhoenix/master

update

josheewa committed:
Merge pull request #6 from Legend-of-iPhoenix/master

update

josheewa committed:
Merge pull request #7 from Legend-of-iPhoenix/master

Update

josheewa committed:
Add files via upload

josheewa committed:
Slowed down the rainbow a tiny bit

josheewa committed:
Delete Red_button.png

josheewa committed:
Add files via upload

josheewa committed:
Added favicon

josheewa committed:
Delete icon.png

josheewa committed:
Add files via upload

josheewa committed:
Fix as needed. it's definitely wrong

Legend-of-iPhoenix committed:
Update index.html

josheewa committed:
Added a github repo link

josheewa committed:
Update index.html

Legend-of-iPhoenix commented:
Keep revising and adding changes to this pr, I have IRL commitments in a few minutes.

I'm probably going to redesign the favicon. I'll post some concepts here later, and we'll debate them. I'm not saying no to the current favicon suggestion designed by @jcgter777 .

josheewa committed:
Maybe? moved down

Legend-of-iPhoenix committed:
Update index.html

josheewa committed:
Update style.css

josheewa committed:
I'm trying to add a title, but when I test, thebutton disappears. Help?

josheewa committed:
Add files via upload

I need to delete #3

josheewa committed:
Delete favicon 3.png

josheewa committed:
Add files via upload

josheewa committed:
Delete favicon 4.png

josheewa committed:
Add files via upload

josheewa committed:
Delete favicon 2.png

josheewa committed:
Add files via upload

josheewa committed:
Update README.md

Legend-of-iPhoenix reviewed (commented):
:)

josheewa commented:
@Legend-of-iPhoenix I need to learn how to call time. 😝 from firebase, that is.

josheewa committed:
Link this to "About" instead of README.md.

josheewa committed:
Linked README.txt instead of the md. 

I used iphoenix's username in place of jcgter777 in the link, so it might not work when tested. (will not, I should say)

josheewa committed:
Update style.css

josheewa committed:
Delete favicon 4.png

josheewa committed:
Update README.txt

josheewa committed:
Update README.md

josheewa committed:
iPhoenix: Can you check if that works? Inspect element makes the thing stay on the loading thing, and I can't refresh without ruining it.

josheewa committed:
Copied over iPhoenix's edits

josheewa committed:
Copied over iPhoenix's edits

josheewa committed:
Re-added my edits

josheewa committed:
Re-added my change

josheewa committed:
Fixed a dumb mistake

josheewa committed:
Set theme jekyll-theme-cayman

josheewa committed:
Fixed cursor flashing, but takes away hover thing

josheewa committed:
Delete _config.yml

josheewa committed:
Update style.css

josheewa committed:
Slowed down rainbow for jonbush

josheewa committed:
Made rainbow faster for jonbush.

josheewa committed:
Made rainbow a bit more reasonable speed. Change as needed

ticalc-travis committed:
Merge remote-tracking branch 'upstream-jcgter777/master' into jcgter777

josheewa committed:
Update style.css

ticalc-travis committed:
Experiment for mobile use

josheewa committed:
changed literally the whole button

josheewa committed:
Merge pull request #8 from ticalc-travis/jcgter777-css-mobile

Jcgter777 css mobile

josheewa committed:
Edited

josheewa committed:
Update README.txt

ticalc-travis committed:
Fix CSS for mobile

josheewa committed:
Merge pull request #9 from ticalc-travis/jcgter777-css-mobile-2

Fix CSS for mobile

josheewa committed:
Fixed a SUPER DUMB mistake tev told me about after mentioning the extra """

josheewa committed:
Update style.css

ticalc-travis committed:
Fix some HTML syntax errors

ticalc-travis committed:
Make title centered

josheewa committed:
Merge pull request #10 from ticalc-travis/jcgter777-html-fixes

Jcgter777 html fixes

123outerme committed:
Try this bg

josheewa committed:
Revert "Jcgter777 html fixes"

josheewa committed:
Merge pull request #11 from jcgter777/revert-10-jcgter777-html-fixes

Revert "Jcgter777 html fixes"

josheewa committed:
Merge pull request #12 from 123outerme/patch-8

Try this bg

josheewa committed:
Revert "Try this bg"

josheewa committed:
Merge pull request #13 from jcgter777/revert-12-patch-8

Revert "Try this bg"

josheewa committed:
Update style.css

ticalc-travis committed:
Update beta to current version

ticalc-travis committed:
Center title and correct HTML syntax errors

josheewa committed:
Merge pull request #14 from ticalc-travis/jcgter777-html-fixes-try2

Jcgter777 html fixes try2

ticalc-travis committed:
Apply beta changes to main project

josheewa committed:
Changed bg color

josheewa committed:
Changed shadow color

josheewa committed:
Merge pull request #16 from jcgter777/jcgter777-patch-1

Changed bg color

josheewa committed:
Update index.html

josheewa committed:
Update index.html

josheewa committed:
Merge pull request #15 from ticalc-travis/jcgter777-html-fixes-try2

Apply beta changes to main project

josheewa committed:
Update style.css

josheewa committed:
Update style.css

josheewa committed:
Update style.css

josheewa committed:
Update style.css

josheewa committed:
Update style.css

Legend-of-iPhoenix committed:
Update style.css

Legend-of-iPhoenix committed:
Update main.js

Legend-of-iPhoenix committed:
Update main.js

Legend-of-iPhoenix committed:
Delete favicon 2.png

Legend-of-iPhoenix committed:
256th commit.

Legend-of-iPhoenix committed:
unneeded.

Legend-of-iPhoenix committed:
This bothered me, for some reason.
